src/main/webapp/assets/modules/common/auto-tagbox.mod.js

```diff
@@ -0,0 +1,37 @@
+techlooper.directive('autoTagbox', function ($timeout) {
+  return {
+    restrict: "E",
+    replace: true,
+    templateUrl: "modules/common/auto-tagbox.tem.html",
+    scope: {
+      tags: "=",
+      config: "="
+    },
+    link: function (scope, element, attr, ctrl) {
```

Comment:
'ctrl' is defined but never used.<br>'attr' is defined but never used.<br>'element' is defined but never used.